gp-includes/things/translation.php

```diff
@@ -485,8 +485,66 @@ public function reject() {
 		$this->set_status( 'rejected' );
 	}
 
+	/**
+	 * Decides whether the status of a translation can be changed to $desired_status.
+	 *
+	 * @param  string $desired_status The desired status.
+	 * @return bool Whether the status can be set.
```

Comment:
* Let's change `$desired_status` in the summary to `a desired status`.
* @param and @return values shouldn't be aligned
* @since tag is missing